### ISSUE TITLE
the result is wrose than that in your paper

### ISSUE THREAD

**Issue by jamesben6688** [jeya-maria-jose/Medical-Transformer#41]
Hello, I run the training and testing on the MoNuSeg dataset using the command you provided (set modelname as MedT). 

The testing ground truth is from 0-255, so I used the following to convert it:

`
gt_arr = np.where(gt_arr>127, 1, 0)
`

Then I compute the f1 score of the val images, which is less than 0.6. Did I do anything wrong?

**Comment by jamesben6688** [comment]
The loss is above 0.2 when trained over 400 epochs.

**Comment by jeya-maria-jose** [comment]
Hi, please use the original training script and not the Colab script for training.
